_data/menu.yml

```diff
@@ -56,8 +56,8 @@
   items:
     - title: 'Introductory Articles'
       url: '/doc/philosophy/intros.html'
-    - title: 'The OMDoc Format'
-      url: '/doc/philosophy/omdoc.html' 
+    - title: 'The OM/doc Format'
```

Comment:
Not sure I like the OM/doc spelling. The name is OMDOC (without any /s)

Reply:
Actually, the name is `OMDoc`.

Reply:
Ah, that was an unintentional mistake of mass regex replacement of `/doc` URLs (relativizing them). Reverted in latest commit.